tests/netebpfext_unit/netebpfext_unit.cpp

```diff
@@ -380,6 +391,11 @@ netebpfext_unit_invoke_sock_addr_program(
         action = client_context->sock_addr_action;
     }
 
+    if (!is_admin) {
```

Comment:
shouldn't 387-392 be inside `if (is_admin)` ?

Reply:
Fixed as per suggestion.